src/components/LargeCard/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Card, Button, Typography } from 'antd';
+import data from './data';
```

Comment:
```suggestion

import data from './data';
```
leave line between import local files and libraries

Reply:
Done!